books/free-programming-books-pt_BR.md

```diff
@@ -275,6 +276,7 @@
 
 ### Kubernetes
 
+* [Descomplicando Kubernetes](https://github.com/badtuxx/DescomplicandoKubernetes/blob/main/SUMMARY.md) - Jeferson Fernando
```

Comment:
What about if url is replaced by

https://livro.descomplicandokubernetes.com.br

It's a bit simple same as docker link ;)

Reply:
Oh, thanks for the tip, i'll change it 🥇

Reply:
> Oh, thanks for the tip, i'll change it 🥇

https://github.com/EbookFoundation/free-programming-books/pull/5527/commits/a353c5e389223bf24a4a51d8b5b6a088abad8e27